### ISSUE TITLE
Display different currencies in multistores

### ISSUE THREAD

**Issue by Schmerge** [smartstoreag/SmartStoreNET#647]
Example: Shop A = Currencie €, Shop B = Currencie $

**Comment by adameldaba** [comment]
We are facing the same issue.  We have multiple stores in different countries using different currencies.  As per the configurations today, we have inserted all our products using AED as the primary currency as the first store is in the UAE.  Now all pricing are AED as primary. Lets call this store UAE_Store

Now I just opened a second store in Egypt which uses EGP as its currency. Lets call this store EGP_Store.

When I select the primary currency as EGP for EGP_Store and select the exchange currency as AED (because all my pricing is based on this currency), and save; when I go to the store EGP_Store, the currency displayed is correct, but the value is incorrect, because it does not exchange the value. It only exchanges the value if I manually select another currency from the top right which is incorrect.  

I don't want the user to keep changing currency, I want depending on which store they enter, the correct currency and the value is displayed to the user.  

A quick fix could be to add a new field: "DisplayCurrency" and that way for each store we can choose the default currency to be displayed.  If it is not the same as the "Primary Currency" then it's value should be equated to the exchange rate and displayed correctly in the store. This way I can use all the products I currently have and not need to insert them in every country.
